From: Marco Martin <notmart@gmail.com>
Date: Thu, 19 Jan 2017 14:25:16 +0100
Subject: it must also be possible for the status to be decreased again

this fixes the autohide panel being broken the first time an applet
like kickoff opens. when it closes again, the status has to be
decreased again
---
 src/plasma/private/containment_p.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plasma/private/containment_p.cpp b/src/plasma/private/containment_p.cpp
index 646c80021..8ba9356a2 100644
--- a/src/plasma/private/containment_p.cpp
+++ b/src/plasma/private/containment_p.cpp
@@ -123,7 +123,7 @@ void ContainmentPrivate::checkStatus(Plasma::Types::ItemStatus appletStatus)
         }
     }
 
-    if (appletStatus > q->status() && appletStatus != Plasma::Types::HiddenStatus) {
+    if (appletStatus != Plasma::Types::HiddenStatus) {
         q->setStatus(appletStatus);
     }
 }
